plugins/poll/assets/javascripts/discourse/components/poll-options.gjs

```diff
@@ -0,0 +1,98 @@
+import Component from "@glimmer/component";
+import { fn } from "@ember/helper";
+import { on } from "@ember/modifier";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import routeAction from "discourse/helpers/route-action";
+import dIcon from "discourse-common/helpers/d-icon";
+import i18n from "discourse-common/helpers/i18n";
+import PollOptionsDropdown from "./poll-options-dropdown";
+
+export default class PollOptionsComponent extends Component {
+  @service currentUser;
+
+  isChosen = (option) => {
+    return this.args.votes.includes(option.id);
+  };
+
+  get classes() {
+    return this.args.isIrv ? "irv-poll-options" : "";
+  }
+
+  @action
+  sendClick(option) {
+    this.args.sendOptionSelect(option);
+  }
+
+  @action
+  sendRank(option, rank = 0) {
+    this.args.sendOptionSelect(option, rank);
+  }
+  <template>
+    <ul class={{this.classes}}>
+      {{#each @options as |option|}}
+        {{#if @isIrv}}
```

Comment:
That would deserve its own component: poll-irv-option

Reply:
this is done, but slightly tweaked naming convention so things stay together in directory.